src/software/networking/BUILD

```diff
@@ -12,6 +10,19 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "threaded_proto_multicast_listener",
+    hdrs = [
+        "threaded_proto_multicast_listener.h",
+        "threaded_proto_multicast_listener.tpp",
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":proto_multicast_listener",
+        "@boost//:asio",
+    ],
+)
+
 cc_library(
     name = "proto_multicast_sender",
```

Comment:
Add a comment about this being private

Reply:
Made the package default to public and marked the targets specifically as private instead (should be more clear that way).